rct229/rulesets/ashrae9012019/section16/section16rule3.py

```diff
@@ -0,0 +1,99 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.ruleset_model_factory import produce_ruleset_model_instance
+from rct229.rulesets.ashrae9012019 import BASELINE_0, PROPOSED, USER
+from rct229.rulesets.ashrae9012019.ruleset_functions.compare_schedules import (
+    compare_schedules,
+)
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all
+
+
+class Section16Rule3(RuleDefinitionListIndexedBase):
+    """Rule 3 of ASHRAE 90.1-2019 Appendix G Section 16 (Elevators)"""
+
+    def __init__(self):
+        super(Section16Rule3, self).__init__(
+            rmds_used=produce_ruleset_model_instance(
+                USER=False,
+                BASELINE_0=True,
+                PROPOSED=True,
+            ),
+            each_rule=Section16Rule3.ElevatorRule(),
+            index_rmd="baseline",
+            id="16-3",
+            description="The elevator cab ventilation fan shall be modeled with the same schedule as the elevator motor.",
+            ruleset_section_title="Elevators",
+            standard_section="Section G3.1",
+            is_primary_rule=True,
+            rmd_context="ruleset_model_descriptions/0",
+            list_path="ruleset_model_descriptions[0].buildings[*].elevators[*]",
+            required_fields={"$": ["calendar"], "$.calendar": ["is_leap_year"]},
+            data_items={"is_leap_year_b": (BASELINE_0, "calendar/is_leap_year")},
+        )
+
+    def is_applicable(self, context, data=None):
+        rmd_p = context.PROPOSED
+
+        return (
+            len(
+                find_all(
+                    "$.ruleset_model_descriptions[0].buildings[*].elevators", rmd_p
+                )
+            )
+            > 0
```

Comment:
I wonder if you can just do `return find_all("$.ruleset_model_descriptions[0].buildings[*].elevators", rmd_p)`

Reply:
Addressed